### PR TITLE
Github Workflow Replacement for Jenkins Jobs, beam_LoadTests_Python_GBK_Flink_Batch; beam_Java_LoadTests_Combine_Smoke_PR

### PR DESCRIPTION
**Pull request by magicgoody**
PR migrates Jenkins Jobs to Github Actions Workflow:
".github/workflows/beam_Java_LoadTests_Combine_Smoke_PR.yml"
".github/workflows/beam_LoadTests_Python_GBK_Flink_Batch.yml"

Jenkins Jobs:
https://ci-beam.apache.org/view/LoadTests/job/beam_Java_LoadTests_Combine_Smoke_PR/
https://ci-beam.apache.org/view/LoadTests/job/beam_LoadTests_Python_GBK_Flink_Batch/

The code has been tested on Self-hosted GKE runners ubuntu 20.04.

Example runs:
beam_Java_LoadTests_Combine_Smoke_PR https://github.com/magicgoody/beam/actions/runs/6451485468
image

Job is disabled in Jenkins.

***

beam_LoadTests_Python_GBK_Flink_Batch.yml
image

image

Test run results in Jenkins, the same as in Github run.
https://ci-beam.apache.org/view/LoadTests/job/beam_LoadTests_Python_GBK_Flink_Batch_PR/

Test works very slow and fails, there is an open issue.
https://github.com/apache/beam/issues/20146

Job is disabled in Jenkins.

***

**FIX FOR beam_PostCommit_XVR_GoUsingJava_Dataflow**

Fixed issue with container name generation on step Task :sdks:go:container:dockerPrepare "sdks\go\test\run_validatesrunner_tests.sh".
```
> Task :sdks:go:container:dockerPrepare
ERROR: invalid tag "us.gcr.io/apache-beam-testing//beam_go_sdk:20231011-063112": invalid reference format
```
https://github.com/apache/beam/actions/runs/6478612217/job/17590667466#step:7:7320

Issue was fixed, however workflow is broken. The same issue as in Jenkins
https://github.com/magicgoody/beam/actions/runs/6482495803
image

https://ci-beam.apache.org/view/PostCommit/job/beam_PostCommit_XVR_GoUsingJava_Dataflow/
image

***

**FIX FOR beam_PerformanceTests_xlang_KafkaIO_Python**

github.event.comment.body == 'Run Python xlang KafkaIO Performance Test'
Trigger phrase was fixed

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: .github/workflows/beam_Java_LoadTests_Combine_Smoke.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      - name: Prepare test arguments
+        uses: ./.github/actions/test-arguments-action
+        with:
+          test-type: load
+          test-language: java
+          argument-file-paths: |
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/java_LoadTests_Combine_Smoke_CombineLoadTest_load_test_Dataflow-1.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/java_LoadTests_Combine_Smoke_CombineLoadTest_load_test_Dataflow-2.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/java_LoadTests_Combine_Smoke_CombineLoadTest_load_test_Dataflow-3.txt
+      - name: run CombineLoadTest load test Dataflow-1
```
Comment (andreydevyatkin) [inline .github/workflows/beam_Java_LoadTests_Combine_Smoke.yml:81]: Please add a comment above:
```
# The env variables are created and populated in the test-arguments-action as "<github.job>_test_arguments_<argument_file_paths_index>"
```

**Review by andreydevyatkin — APPROVED** [review]
LGTM, just had one minor comment

File: .github/workflows/beam_Java_LoadTests_Combine_Smoke.yml (hunk 1 above)
Comment (magicgoody) [reply .github/workflows/beam_Java_LoadTests_Combine_Smoke.yml:82]: fixed, thanks

File: .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml [hunk 2]
```diff
@@ -73,6 +73,9 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: '3.8'
+      # username is required to generate container name in step \sdks\go\test\run_validatesrunner_tests.sh
+      - name: Get current user
+        run: echo "USER="$(whoami)"" >> $GITHUB_ENV
       - name: run XVR GoUsingJava Dataflow script
         env:
           CLOUDSDK_CONFIG: ${{ env.KUBELET_GCLOUD_CONFIG_PATH}}
```
Comment (andreydevyatkin) [inline .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml:81]: Do we really need to have this step? Can we just use the same approach as for other workflows:
```suggestion
      - name: run XVR GoUsingJava Dataflow script
        env:
          USER: github-actions
          CLOUDSDK_CONFIG: ${{ env.KUBELET_GCLOUD_CONFIG_PATH}}
```

**Review by andreydevyatkin — CHANGES_REQUESTED** [review]
Having one comment that I think should be fixed

File: .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml (hunk 2 above)
Comment (magicgoody) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: thanks, updated

**Review by andreydevyatkin — APPROVED** [review]
Thanks!

**Pull request merged**